backend/blueprint/api/run.py

```diff
@@ -26,6 +27,7 @@
 app.register_blueprint(homePage.router)
 app.register_blueprint(errorHandlers.router)
 app.register_blueprint(resetUser.router, url_prefix='/reset')
+app.register_blueprint(users.router, url_prefix='/admin')
```

Comment:
This should not be admin endpoint as it is clearly seen in the JS. It should be `/user` according to me.

Reply:
please see the issue carefully

Reply:
Why will the admin will create user and delete user??? Why he needs the details of all the user? Is it a product or anlytical tool? Are you clear on your point?